main.js

```diff
@@ -92,12 +107,13 @@ function ftTime(el) {
 
 	if (!date) return;
 
-	var interval = Math.round(((new Date()) - date) / 1000);
-	printer.innerHTML = interval < (365 * 60 * 60 * 24) ? timeAgo(date, interval) : format(date, 'date');
+	printer.innerHTML = ftTime(date);
+
 	el.title = format(date, 'datetime');
 	el.setAttribute('data-o-date-js', '');
 }
 
+
 function timeAgo (date, interval) {
```

Comment:
It would probably be a major release, but I don't think that's an issue. Or, as you've already released this, you could make another pr cleaning that up and we can make a major with that